src/main/scala/firrtl/Visitor.scala

```diff
@@ -60,7 +60,7 @@ class Visitor(infoMode: InfoMode) extends AbstractParseTreeVisitor[FirrtlNode] w
 
   private def visitInfo(ctx: Option[InfoContext], parentCtx: ParserRuleContext): Info = {
     def genInfo(filename: String): String =
-      stripPath(filename) + "@" + parentCtx.getStart.getLine + "." +
+      stripPath(filename) + " " + parentCtx.getStart.getLine + "." +
```

Comment:
Might as well take this opportunity to make it consistent with the Chisel source locators (unless you think there's value in distinguishing `:` for Chisel and `.` for FIRRTL).

```suggestion
      stripPath(filename) + " " + parentCtx.getStart.getLine + ":" +
```